app/models/merchant.rb

```diff
@@ -14,4 +14,13 @@ def self.most_revenue(quantity)
     .limit(quantity)
   end
 
+  def self.most_items(quantity)
+    quantity = quantity.to_i
+    joins(invoices: [:invoice_items, :transactions])
+    .select('merchants.*, sum(invoice_items.quantity) as total_sold')
+    .group(:id)
+    .order(total_sold: :desc)
+    .limit(quantity)
```

Comment:
@BrianZanti @dionew1, I need a quick clarification. Is the most_items sold query supposed to include or exclude failed invoices? I included failed invoices because I assumed that the items have still been sold.